dev/io.openliberty.microprofile.telemetry.2.0.internal/src/io/openliberty/microprofile/telemetry20/internal/config/OpenTelemetryVersionedConfigurationImpl.java

```diff
@@ -64,4 +76,61 @@ private Map<String, String> getTelemetryPropertyDefaults() {
         Map<String, String> telemetryProperties = new HashMap<String, String>();
         return telemetryProperties;
     }
+
+    @Override
+    public OpenTelemetry createServerOpenTelemetryInfo(HashMap<String, String> telemetryProperties) {
+        try {
+            String otelInstanceName = "io.openliberty.microprofile.telemetry.runtime";
+
+            if (AgentDetection.isAgentActive()) {
+                // If we're using the agent, it will have set GlobalOpenTelemetry and we must use its instance
+                // all config is handled by the agent in this case
+                return GlobalOpenTelemetry.get();
+            }
+
+            ClassLoader classLoader = OpenTelemetry.noop().getClass().getClassLoader();
+
+            //Builds tracer provider if user has enabled tracing aspects with config properties
+            if (!checkDisabled(telemetryProperties)) {
+                OpenTelemetry openTelemetry = AccessController.doPrivileged((PrivilegedAction<OpenTelemetry>) () -> {
+                    return buildOpenTelemetry(telemetryProperties, OpenTelemetryVersionedConfigurationImpl::customizeResource, classLoader);
+                });
+
+                if (openTelemetry != null) {
+                    System.out.println("Creating Server instance!");
```

Comment:
Convert to Tr?

Reply:
We already have the otel instance being returned in https://opentelemetryinfofactoryimpl.java/

Removed it for now but we can add it later if still needed.